include/graphblas/algorithms/norm.hpp

```diff
@@ -36,13 +36,80 @@
 
 #include <graphblas.hpp>
 
+//#include "../base/internalops.hpp"
+
+
 #include <cmath> // for std::sqrt
 
 
 namespace grb {
 
+	namespace internal {
+
+		template<
+			typename IN1, typename IN2, typename OUT,
+			enum Backend implementation = config::default_backend
+		>
+		class conjuagte_mul {
```

Comment:
This operator is a very elegant solution. Are there any concerns with exposing it in the standard `ops.hpp` header, i.e., making it free for anyone to use? (At first sight, I would prefer that!)

Reply:
I believe that it can be exposed in the ops.hpp, it is quite common operator in the complex field.

Reply:
@anyzelman  Actually, there is one technical problem: `conjugate_mul` depends on `grb::utils::is_complex`, so ether will internalops.hpp_  include _utils/iscomplex.hpp_ or `is_complex` should be moved somewhere else, where it is more natural for _internalops.hpp_ to include from?
If the current solution is ok for you please free to merge, otherwise let me know your preferences. 
I have rebased and pushed the latest version.

Reply:
Now I see that _internalops.hpp_ already includes _utils/suppressions.h_ so I will move `conjugate_mul`  as well.

Reply:
done!